src/omnisharp/server.ts

```diff
@@ -235,20 +235,17 @@ export class OmniSharpServer {
 
         const solutionPath = launchTarget.target;
         const cwd = path.dirname(solutionPath);
+        this._options = Options.Read();
+        
         let args = [
             '-s', solutionPath,
             '--hostPID', process.pid.toString(),
             '--stdio',
             'DotNet:enablePackageRestore=false',
-            '--encoding', 'utf-8'
+            '--encoding', 'utf-8',
+            '-loglevel', this._options.loggingLevel
```

Comment:
Should this be `--loglevel'? If not, we added the option incorrectly in OmniSharp. Only short arguments should have a single dash. If we did it incorrectly, could we fix that up in omnisharp-roslyn?